api/tests/routes/backoffice/venues_test.py

```diff
@@ -198,6 +201,34 @@ def test_list_venues_by_department(self, authenticated_client, venues):
         assert len(rows) == 1
         assert int(rows[0]["ID"]) == venues[0].id
 
+    def test_list_venues_by_id(self, authenticated_client, venues):
+        with assert_num_queries(self.expected_num_queries):
+            response = authenticated_client.get(url_for(self.endpoint, q=42))
+            assert response.status_code == 200
+
+        rows = html_parser.extract_table_rows(response.data)
+        assert len(rows) == 1
+        assert int(rows[0]["ID"]) == venues[0].id
+
+    def test_list_venue_by_multiple_ids(self, authenticated_client, venues):
+        with assert_num_queries(self.expected_num_queries):
+            response = authenticated_client.get(url_for(self.endpoint, q="42, 43"))
+            assert response.status_code == 200
+
+        rows = html_parser.extract_table_rows(response.data)
+        assert len(rows) == 2
+        assert int(rows[0]["ID"]) == venues[1].id
+        assert int(rows[1]["ID"]) == venues[0].id
```

Comment:
L'ordre est-il déterministe dans la liste affichée ? Si non, mieux vaut éviter le test _flaky_ en vérifiant l'ensemble des ids sans ordre : 
```suggestion
        assert {int(row["ID"]) for row in rows} == {venues[1].id, venues[0].id}
```